apps/docs/components/mdx-components/code.tsx

```diff
@@ -5,12 +5,7 @@ import { useTheme } from 'next-themes'
 import { useEffect, useState } from 'react'
 
 export const Code = (props: any) => {
-	if (!props.className) {
-		return <code {...props} />
-	}
-
-	const language = props.className.replace('language-', '')
-	return <CodeBlock code={{ [`App.${language}`]: props.children.trim() }} />
+	return <code {...props} />
```

Comment:
why no more sandpack?

Reply:
Sandpack can't server render and it loads an instance of codemirror (in readonly mode) for every code block. I was seeing big content shifts on links to headers as the code blocks loaded and shifted content. We still use it for the examples (because of the multi-page tabs, which are nice) but from what I can tell it's not meant for the way we were using it.